website/docs/for-capacitor/overview.md

```diff
@@ -27,33 +27,31 @@ Federated Capacitor complements existing micro frontend tools like Module Federa
 
 ## Getting Started
 
-To begin with Federated Capacitor you will need to [signup for a key.](../getting-started)
+:::caution
+Add docs about how to get a registry key?
```

Comment:
Left this here for a discussion point. What did we have for the docs before we added the ability to use the Portals key?